coralproject/talk
# [CORL-748] Refactor v2 css variable names into post css variables

## What does this PR do?

Refactor v2 css variables out of the `variables2.ts` file into post css variables in the css files.

We don't want to clog up the variables list with every design token from AirTable. Instead, more efficient to use Post CSS var's that compile down into the CSS bundle more efficiently.

This was a suggestion from Kiwi part-way through our branding changes that I apparently never got around to migrating. This remedies this before it becomes a mess and we forget we needed to move these out of `variables2.ts`.

## How do I test this PR?

- Build and run
- Test to see that the moderation area isn't borked from styles hopping to new location
